controllers/events.js

```diff
@@ -34,14 +34,13 @@ exports.signUpRSVP = async (req, res, next) => {
     const lst = event.userList;
 
     // eslint-disable-next-line no-underscore-dangle
```

Comment:
For these rules, if you don't want them you can chuck them into the ESLint config file. I can help with that later haha

Reply:
Haha sure